packages/shared/src/components/squads/Details.tsx

```diff
@@ -170,6 +171,28 @@ export function SquadDetails({
               }}
             />
           )}
+          <div className="flex flex-col justify-start">
+            <h4 className="mb-2 font-bold typo-headline text-theme-label-primary">
+              Post permissions
+            </h4>
+            <p className="mb-6 text-salt-90 typo-callout">
+              Choose who is allowed to post new content in this Squad.
+            </p>
+            <RadioItem
+              name="permissionLevel"
+              value="all-members"
+              checked={false}
+            >
+              All members
+            </RadioItem>
+            <RadioItem
+              name="permissionLevel"
+              value="only-moderators"
```

Comment:
I think @sshanzel suggested making this a boolean for instance as `restrictMemberPosting`.
So with `onlyModerators` it could be "true" and for all member "false"

Think it makes sense, let me know what you think.